src/main/java/com/synopsys/integration/detect/workflow/blackduck/font/DetectFontLocator.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * synopsys-detect
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.detect.workflow.blackduck.font;
+
+import java.io.File;
+
+import com.synopsys.integration.detect.configuration.DetectUserFriendlyException;
+
+public interface DetectFontLocator {
+    String FONT_FILE_NAME_REGULAR = "NotoSansCJKtc-Regular.ttf";
```

Comment:
If the names are hardcoded - does this mean a user can't supply their own font in the folder? They have to follow our naming pattern? I thought we were going to try to dynamically load from a folder.

Reply:
We actually need the risk report generator in blackduck-common to change to support loading in multiple fonts. I would make that change in a separate improvement.